docs/tidb_http_api:scan MVCC Information with a start-key (#49010)
pingcap/tidb: docs/tidb_http_api.md

    }
    ```

>     *Hint: List MVCC Information of the keys startted with a specified handle ID:*

Reviewer: ```suggestion
    *Hint: The method to convert the Hex format key returned by TiDB API into the format recognized by [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control).*
```

>     step 1: Get the hex of the start key from tidb.

Reviewer 2: ```suggestion
    Step 1: Get the hex of the key you need. For example you may find the key from the `mvcc` HTTP API..
```
If we are talking about

> The method to convert the Hex format key returned by TiDB API into the format recognized by tikv-ctl

then it's not the *start key*, it's just *a key*.

>      ```shell

Reviewer 2: Please add empty lines before and after code blocks

        $curl http://127.0.0.1:10080/mvcc/key/test/t1/1
        {
        "key": "7480000000000008C65F728000000000000001",
        "region_id": 10,
        "value": {
            "info": {
                "writes": [
                    {
                        "start_ts": 445971968923271174,
                        "commit_ts": 445971968923271175,
                        "short_value": "gAACAAAAAgMIAAkAc2hpcmx5YTQi"
                    },
                    {
                        "start_ts": 445971959499980803,
                        "commit_ts": 445971959499980804,
                        "short_value": "gAACAAAAAgMIAAkAc2hpcmx5YTQL"
                    }
                ]
            }
        }
        }
     ```

>      step 2: convert the hex key from tidb to escaped key with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)

Reviewer 2: ```suggestion
     Step 2: Convert the hex key from tidb to escaped format with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)
```

Reviewer 2: Also capitalize the first latter of other sentences below.

     ```shell
      ./tikv-ctl --to-escaped '7480000000000008C65F728000000000000001'
      t\200\000\000\000\000\000\010\306_r\200\000\000\000\000\000\000\001
     ```
     step 3: Encode the key to make it memcomparable in tikv with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)
     ```
     ./tikv-ctl --encode 't\200\000\000\000\000\000\010\306_r\200\000\000\000\000\000\000\001'
     7480000000000008FFC65F728000000000FF0000010000000000FA
     ```

>      step 4: convert the hex key to escaped key since `tikv-ctl scan` only accepts escaped key.

Reviewer 2: ```suggestion
     step 4: convert the key to escaped format again since `tikv-ctl scan` only accepts escaped key, but the `--encode` command outputs the key in hex format.
```

     ```

>      ./tikv-ctl --to-escaped    '7480000000000008FFC65F728000000000FF0000010000000000FA'

Reviewer: ```suggestion
     ./tikv-ctl --to-escaped '7480000000000008FFC65F728000000000FF0000010000000000FA'
```

     t\200\000\000\000\000\000\010\377\306_r\200\000\000\000\000\377\000\000\001\000\000\000\000\000\372
     ```

>      step 5: Add the prefix "z" as the start-key to scan from tikv with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)

Reviewer 2: ```suggestion
     Step 5: Add a prefix "z" to the key. Then the key can be used in other commands of [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control). For example, use the following command to scan from tikv.
```

     ```

>      ./tikv-ctl  --host "127.0.0.1:20162" scan --from 'zt\200\000\000\000\000\000\010\377\306_r\200\000\000\000\000\377\000\000\001\000\000\000\000\000\372' --limit 5 --show-cf write,lock,default 

Reviewer: It's better to describe which host is it, or which IP/Port should be used.

    }
    ```

    *Hint: The method to convert the Hex format key returned by TiDB API into the format recognized by [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control).*
    

>     Step 1: Get the hex of the key you need. For example you could find the key by the following TiDB  API.

Reviewer 2: ```suggestion
    Step 1: Get the hex of the key you need. For example you could find the key by the following TiDB API.
```

     ```shell
        $curl http://127.0.0.1:10080/mvcc/key/test/t1/1
        {
        "key": "7480000000000008C65F728000000000000001",
        "region_id": 10,
        "value": {
            "info": {
                "writes": [
                    {
                        "start_ts": 445971968923271174,
                        "commit_ts": 445971968923271175,
                        "short_value": "gAACAAAAAgMIAAkAc2hpcmx5YTQi"
                    },
                    {
                        "start_ts": 445971959499980803,
                        "commit_ts": 445971959499980804,
                        "short_value": "gAACAAAAAgMIAAkAc2hpcmx5YTQL"
                    }
                ]
            }
        }
        }
     ```

>      Step 2: Convert the hex format from tidb to escaped format with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)

Reviewer 2: ```suggestion
     Step 2: Convert the key from hex format to escaped format with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)
```

     ```shell
      ./tikv-ctl --to-escaped '7480000000000008C65F728000000000000001'
      t\200\000\000\000\000\000\010\306_r\200\000\000\000\000\000\000\001
     ```

     Step 3: Encode the key to make it memcomparable in tikv with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)

     ```
     ./tikv-ctl --encode 't\200\000\000\000\000\000\010\306_r\200\000\000\000\000\000\000\001'
     7480000000000008FFC65F728000000000FF0000010000000000FA
     ```

>      Step 4: Convert the hex format to escaped format again since most `tikv-ctl` commands only accept keys in escaped format while the `--encode` command outputs the key in hex format.

Reviewer 2: ```suggestion
     Step 4: Convert the key from hex format to escaped format again since most `tikv-ctl` commands only accept keys in escaped format while the `--encode` command outputs the key in hex format.
```
